Dockerfile_RISCV

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu:18.04
+RUN apt-get update 
+RUN apt-get install -y build-essential git curl gawk texinfo expat \
+    libexpat1-dev bison flex libz-dev python autoconf
+# Prepare to download and compile RISCV toolchain
+ENV HOME_DIR=/home/root
+ENV RISCV_SRC_DIR=$HOME_DIR/riscv-gnu-toolchain
+ENV RISCV_DIR=$HOME_DIR/riscv-tools
+RUN mkdir -p $HOME_DIR
+WORKDIR $HOME_DIR
+RUN git clone --recursive https://github.com/riscv/riscv-gnu-toolchain -j $(nproc)
+WORKDIR $RISCV_SRC_DIR
+# Checkout a specific hash to keep the output binaries consistent
+RUN git checkout 23a038856764808d75b6afe96f649980609ae4c6 
```

Comment:
Sorry for the late comment... can you checkout a tag instead of a hash? Are there tags for specific versions of gcc?

Reply:
They have tags, each has a specific version of gcc.